### ISSUE TITLE
[ Check-box-Disabled-Won't Get Focus ] : Focus won't go on Disabled check box, So It is not convey message to  Narrator/NVDA .

### ISSUE THREAD

**Issue by v-jashr** [microsoft/fluentui#13704]
Actual Behavior:
In Disabled option checkbox won't get focus. 

Repro: https://developer.microsoft.com/en-us/fabric#/controls/web/checkbox

Repro Step:
Open Edge browser with above link.
Go to checkbox, tab through page try to reach on disabled checkbox.
Now observe that it's difficult to focus on checkbox which is disabled.

Expected 
Focus should go on disabled check box and narrator should speak it is disabled.

Can you please let us know is any property available for this . As on official web i am not able see.

**Comment by paulgildea** [comment]
@jurokapsiar Can you help on the pattern we are supposed to use here?

It seems like we're supposed to use `aria-disabled` but just using that allows you to interact with the checkbox?

**Comment by jurokapsiar** [comment]
Browser by default does not focus disabled elements.
Some applications or experiences might decide to make the disabled elements focusable, typically by not using `disabled` prop but by using just `aria-disabled` and ignoring all events (click, key...).

This is the case for all other Fabric components, so I assume it would make sense if Checkbox followed the pattern.

**Comment by xugao** [comment]
Thanks @jurokapsiar! I think we should make disabled checkbox focusable. we currently lack of consistency across all components on this point. But i think for the long term, we should make all disabled control focusable with `aria-disabled=true`.

cc @betrue-final-final

**Comment by xugao** [comment]
@v-jashr - checked internally with our team members and this is by design. we don't want to enable the focus which sacrifices efficiency for keyboard navigation. User can still get the information on the disabled item using scan/browse mode.
